frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -986,14 +995,17 @@ pub mod pallet {
 
 			// create the submission
 			let deposit = Self::deposit_for(&raw_solution, size);
-			let reward = {
+			let call_fee = {
 				let call = Call::submit { raw_solution: raw_solution.clone() };
-				let call_fee = T::EstimateCallFee::estimate_call_fee(&call, None.into());
-				T::SignedRewardBase::get().saturating_add(call_fee)
```

Comment:
dq: why is this removed?

Reply:
We used to calculate the reward here, but now we just want the call fee. This is because we now calculate the reward on the spot when paying it out, but we can't calculate the call fee when paying it out because it is highly variable.

The reward is `T::SignedRewardBase::get().saturating_add(call_fee)`

On the other hand, to get the call fee we just need `T::EstimateCallFee::estimate_call_fee(&call, None.into())`.